distribution/com.modrinth.ModrinthApp.desktop

```
@@ -0,0 +1,12 @@
+[Desktop Entry]
+Name=Modrinth App
+Comment=Modrinth's future game launcher
```

```suggestion
Comment=Modrinth's game launcher for creating and playing Modrinth projects
```
I don't know too much about packaging here, but this should probably be synced with the metainfo (but I could be wrong!)

Also, GH thinks the rest of this file past the `'` in `Modrinth's` is one string, is that an issue?

---

not a requirement, but still a nice catch. thanks! :p

> Also, GH thinks the rest of this file past the ' in Modrinth's is one string, is that an issue?

and no, that's a github error